Flick/Controllers/ListViewController.swift

```diff
@@ -34,21 +34,26 @@ class ListViewController: UIViewController {
     // MARK: - Private Data Vars
     private let cellPadding: CGFloat = 20
     private let edgeInsets: CGFloat = 28
+    private let listId: Int
     private var listSummaryHeight: CGFloat = 80
-    private var list: MediaList!
+    private var list: MediaList?
```

Comment:
What happens to the screen if list doesn't have a value?

Reply:
the view first shows up like this when it doesn't have a value for list. 
Once we get the list from backend, the list info is populated, it's pretty fast, takes like one sec
![Simulator Screen Shot - iPhone 11 Pro - 2020-08-19 at 21 22 10](https://user-images.githubusercontent.com/48968041/90705987-272aa500-e262-11ea-8bb7-3f45459add5f.png)